ifsbench/validation/frame_close_validation.py

```diff
@@ -0,0 +1,77 @@
+# (C) Copyright 2020- ECMWF.
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+
+from dataclasses import dataclass
+from typing import Any, List, Tuple
+
+import numpy
+from pandas import DataFrame
+
+from ifsbench.validation.frame_util import get_float_columns
+
+@dataclass
+class FrameCloseValidation:
```

Comment:
Does this really need a class? It seems easier to just have the compare function and pass in rtol / atol with 0 as defaults.

Reply:
slight preference to use the simpler pure function approach until we know a class is needed (that also then maybe inherits from something so has additional requirements on the API). But also ok to leave it since you have it.